converters/toZigbee.js

```diff
@@ -2528,6 +2528,32 @@ const converters = {
             ], manufacturerOptions.ubisys));
         },
     },
+    ubisys_dimmer_setup: {
+        key: ['ubisys_dimmer_setup'],
```

Comment:
I propose to chnage this to `ubisys_dimmer_mode` so we don't have a nested object. This allows it to add it to exposes, (as enum, see example here: https://github.com/Koenkk/zigbee-herdsman-converters/blob/38752abc80c9af8459d2a96977fc20b92009450a/devices.js#L735)

Reply:
Good idea, will try it (new to javascript ;-))